docsbox/docs/views.py

```diff
@@ -38,22 +40,27 @@ def post(self, file_id):
         """
         if request.files and "file" in request.files:
             mimetype = create_tmp_file_and_get_mimetype(request.files["file"], None, schedule_file_del=False)['mimetype']
-        elif file_id:
-            r = get_file_from_via(file_id)
+        elif file_id and bool(re.match(r"([0-f]{8}-[0-f]{4}-[0-f]{4}-[0-f]{4}-[0-f]{12})", file_id)):
```

Comment:
Since this regex is used in two locations and is not that self-explanatory, put it in a function and use a descriptive name for it.

Reply:
created new function 'is_valid_uuid' in utils module